data/initMPS.sql

```diff
@@ -14,6 +14,8 @@ CREATE TABLE IF NOT EXISTS devices(
       connectionstatus boolean,
       mpsusername text,
       tenantid varchar(36),
+      friendlyhostname varchar(256),
+      friendlydnssuffix varchar(256),
```

Comment:
rename to `dnssuffix`, use camlCase for  column rename in the query

Reply:
Hi Mike. I googled and see that usually database entries to be in small case. We can have a sync-up if you dont agree.
https://www.sisense.com/blog/better-sql-schema/